src/main/resources/assets/js/components/AppListComponent.jsx

```diff
@@ -99,6 +100,11 @@ var AppListComponent = React.createClass({
               </span>
             </th>
             <th className="text-right">
+              <span className={headerClassSet}>
```

Comment:
It could be a function that returns a binary number that representing the state of the app as a whole
```
+-----------+---------------+--------+---------+---------+-------------+
| unhealthy | over-capacity | staged | healthy | running | unscheduled |
+-----------+---------------+--------+---------+---------+-------------+
```
So an app with unhealthy and healthy tasks would return:
```
+-----------+---------------+--------+---------+---------+-------------+
|     1     |      0        |    0   |    1    |    0    |      0      | = 36
+-----------+---------------+--------+---------+---------+-------------+
```
and an app with over-capacity tasks and running tasks would give:
So an app with unhealthy and healthy tasks would return:
```
+-----------+---------------+--------+---------+---------+-------------+
|     0     |      1        |    0   |    0    |    1    |      0      | = 18
+-----------+---------------+--------+---------+---------+-------------+
```